Client/Frontend/Login Management/AddCredentialViewController.swift

```diff
@@ -26,8 +26,7 @@ class AddCredentialViewController: UIViewController {
         tableView.delegate = self
         tableView.dataSource = self
         tableView.estimatedRowHeight = 44.0
-            
-        // Add empty footer view to prevent seperators from being drawn past the last item.
```

Comment:
Could we keep this comment on the footer explanation 🙏 Maybe if we keep it we can fix the typo though `seperators` -> `separators`